liana-gui/src/app/state/buysell.rs

```diff
@@ -510,94 +683,541 @@ impl State for BuySellPanel {
                     .update(action)
                     .map(map_webview_message_static);
             }
-            #[cfg(feature = "webview")]
             BuySellMessage::WebviewAction(action) => {
                 return self
                     .webview
                     .get_or_insert_with(init_webview)
                     .update(action)
                     .map(map_webview_message_static);
             }
-            #[cfg(feature = "webview")]
             BuySellMessage::WebviewOpenUrl(url) => {
                 // Load URL into Ultralight webview
-                tracing::info!("🌐 [LIANA] Loading Ultralight webview with URL: {}", url);
+                if cfg!(debug_assertions) {
+                    tracing::info!("🌐 [LIANA] Loading Ultralight webview with URL: {}", url);
```

Comment:
No need to use `cfg!(debug_assertions)` tracing::log already skips logging on release builds